### DISCUSSION TITLE
Remove the "Show" password option?

### DISCUSSION THREAD

**Discussion by KWebber99** [alphagov/govuk-design-system#4033] [WCAG 2.2]
I work for the IT Infrastructure Team of an accountancy, and as such we have 3 HMRC accounts which are simultaneously logged in by hundreds of members of staff. To mitigate the passwords being shared or leaked, we use a number of browser policies and 3rd party tools that will automatically enter the details for them without them ever needing to see/know the password. Meaning that we don't need to change the password every time someone leaves the firm or be worried about someone copying the password and using it from an unauthorised device.

The "Show" password button that appears to be embedded into the log in page at https://www.access.service.gov.uk/login/signin/creds has undone all that hard work and drastically elevated the risk of the details being leaked. Is there a way to remove the "Show" password button?

Regards, 
Kyle

**Comment by querkmachine** [comment]
Hi @KWebber99,

Sorry for the very late response on this. We don't usually provide support over Discussions, so this probably got missed. Additionally, this wouldn't be something we could help you with, as we don't manage Government Gateway.

Try [getting in contact with HMRC online services](https://www.gov.uk/government/organisations/hm-revenue-customs/contact/complain-about-hmrc-online-services) to see if they have any suggestions, although it's likely that any changes specific to your organisation will have to take place from within your organisation.
